todo-app/db/connection.ts

```diff
@@ -0,0 +1,11 @@
+import { drizzle } from "drizzle-orm/postgres-js";
+import postgres from "postgres";
+import { connectionString } from "./const";
+
+if (connectionString === undefined) {
+  throw new Error("unknown connectionstring");
```

Comment:
better error messages
```suggestion
  throw new Error("database url is not defined");
```